src/universal/conf/private-testnet.conf

```diff
@@ -0,0 +1,88 @@
+# Sample configuration for a custom private testnet.
+mantis {
+  sync {
+    # Whether to enable fast-sync
+    do-fast-sync = false
+
+    # Duration for blacklisting a peer. Blacklisting reason include: invalid response from peer, response time-out, etc.
+    # 0 value is a valid duration and it will disable blacklisting completely (which can be useful when all nodes are
+    # are controlled by a single party, eg. private networks)
+    blacklist-duration = 0
+
+    # Set to false to disable broadcasting the NewBlockHashes message, as its usefulness is debatable,
+    # especially in the context of private networks
+    broadcast-new-block-hashes = false
+  }
+
+  blockchain {
+    # DAO fork configuration (Ethereum HF/Classic split)
+    # https://blog.ethereum.org/2016/07/20/hard-fork-completed/
+    dao = null
+
+    # Custom genesis JSON file
+    # null value indicates using default genesis definition that matches the main network
+    custom-genesis-file = "conf/private-genesis.json"
+  }
+
+  network {
+    discovery {
+      # We assume a fixed cluster, so `bootstrap-nodes` must not be empty
+      discovery-enabled = false
+
+      # Listening interface for discovery protocol
+      interface = ${mantis.network.server-address.interface}
+
+      # Listening port for discovery protocol
+      # port = 30303
+
+      # Set of initial nodes
+      bootstrap-nodes = [
+        "enode://18a551bee469c2e02de660ab01dede06503c986f6b8520cb5a65ad122df88b17b285e3fef09a40a0d44f99e014f8616cf1ebc2e094f96c6e09e2f390f5d34857@127.0.0.1:30303",
+      ]
```

Comment:
Minor, as it will have to be modified anyway, but wouldn't we prefer empty list or even:

```
bootstrap-nodes = [
  # fill in with node URI for the private network, e.g.:
  # "enode://18a551bee469c2e02de660ab01dede06503c986f6b8520cb5a65ad122df88b17b285e3fef09a40a0d44f99e014f8616cf1ebc2e094f96c6e09e2f390f5d34857@127.0.0.1:30303"
]
```